driver/pparse.mli

```diff
@@ -46,20 +46,20 @@ val apply_rewriters: ?restore:bool -> tool_name:string ->
   (** If [restore = true] (the default), cookies set by external
       rewriters will be kept for later calls. *)
 
-val apply_rewriters_str:
-  ?restore:bool -> tool_name:string -> Parsetree.structure ->
-  Parsetree.structure
-val apply_rewriters_sig:
-  ?restore:bool -> tool_name:string -> Parsetree.signature ->
-  Parsetree.signature
+val apply_rewriters_impl:
+  ?restore:bool -> tool_name:string -> Parsetree.implementation ->
+  Parsetree.implementation
+val apply_rewriters_intf:
+  ?restore:bool -> tool_name:string -> Parsetree.interface ->
+  Parsetree.interface
```

Comment:
I would rather keep this part unchanged. I am also not sure that there is a need to carry location information in these rewriting functions: I would expect location information to be passed anyway through the ppx context. But maybe the refine types could help for something (do you know?), yet I would still be in favor of not changing this for now to have a smaller change and not have to worry about breaking user code. We could change this in the future if a need is clarified.